packages/tailwind/src/utils/css/make-inline-styles-for.ts

```diff
@@ -0,0 +1,61 @@
+import selectorParser from "postcss-selector-parser";
+import type { Root, Rule } from "postcss";
+import { unescapeClass } from "../compatibility/unescape-class";
+import { convertCssPropertyToReactProperty } from "../compatibility/convert-css-property-to-react-property";
+
+const walkInlinableRules = (root: Root, callback: (rule: Rule) => void) => {
+  root.walkRules((rule) => {
+    if (rule.parent?.type === "atrule") {
+      return;
+    }
+
+    selectorParser((selector) => {
+      let hasPseudoSelectors = false as boolean;
```

Comment:
```suggestion
      let hasPseudoSelectors = false;
```

Reply:
The problem here is linting. If we use a proper type for the variable eslint complains about types that can be inferred, and if we don't cast, eslint complains that the condition down below is unnecessary, when it actually is necessary.

https://github.com/resend/react-email/blob/560c789acca66d1a7bc68529443efdc69a87cd98/packages/tailwind/src/utils/css/make-inline-styles-for.ts#L18-L20

The best option I found to not disable linting here was to cast the type.